File: lib/src/main/java/com/darklycoder/wifitool/lib/info/action/WiFiRemoveAction.java
package com.darklycoder.wifitool.lib.info.action;

import android.support.annotation.NonNull;

import com.darklycoder.wifitool.lib.interfaces.RemoveWiFiActionListener;

/**
 * 移除
 */
public class WiFiRemoveAction extends IWiFiAction {

    public String SSID;
    public RemoveWiFiActionListener listener;

    public WiFiRemoveAction(String SSID, RemoveWiFiActionListener listener) {
        this.SSID = SSID;
        this.listener = listener;
    }

    @NonNull
    @Override
    public String toString() {
        return super.toString() + " | " + SSID;
    }

}

File: lib/src/main/java/com/darklycoder/wifitool/lib/info/action/WiFiRemoveAction.kt
package com.darklycoder.wifitool.lib.info.action

import com.darklycoder.wifitool.lib.interfaces.RemoveWiFiActionListener

/**
 * 移除
 */
class WiFiRemoveAction(
        var SSID: String,
        var listener: RemoveWiFiActionListener? = null

) : IWiFiAction() {

    override fun toString(): String {
        return "${super.toString()} | " + SSID
    }

}
